index.d.ts

```diff
@@ -111,10 +116,37 @@ declare module "replicate" {
       get: string;
       cancel: string;
       stream?: string;
+      web?: string;
     };
   }
 
-  export type Training = Prediction;
```

Comment:
@zeke Do we want to move to training being a standalone type? Currently it mirrors `Prediction`, so I'm not sure what this will do, if anything, to linting.

Reply:
Yeah, fair! I'm not sure. 😔 I wouldn't be surprised if there were inconsistencies there. Good to clean it up.